### PR TITLE
Lint the Keras - Wasserstein GAN (WGAN) with Gradient Penalty (GP) example

### PR DESCRIPTION
**Pull request by 8bitmp3**
Spotted a few tiny things. Thanks `keras-team` and @fchollet

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thank for the PR! Please replicate the fixes in the corresponding `.ipynb` and `.md` files.

**Comment by 8bitmp3** [comment]
@fchollet Manually linted the Jupyter, Python and Markdown files. Thank you 👍

File: examples/generative/ipynb/wgan_gp.ipynb [hunk 1]
```diff
@@ -486,53 +486,53 @@
    },
    "outputs": [],
    "source": [
-    "# Optimizer for both the networks\n",
-    "# learning_rate=0.0002, beta_1=0.5 are recommened\n",
+    "# Instatiate the optimizer for both the networks\n",
```
Comment (fchollet) [inline examples/generative/ipynb/wgan_gp.ipynb:489]: Instantiate

Comment (8bitmp3) [reply examples/generative/ipynb/wgan_gp.ipynb]: Thanks @fchollet -> `Instantiate the optimizer for both networks` (~~the~~)

**Review by fchollet — APPROVED** [review]
